help.md

```diff
@@ -23,6 +23,10 @@ Look to see if the current bug or feature exists on the [Pulp issues tracker](ht
 
 See bugs confirmed this week and what versions they affect with the [Weekly Bug Report](https://pulp.plan.io/projects/pulp/wiki/Weekly_Bug_Trends).
 
+## GitHub Discussions
+
+We have enabled [GitHub Discussions](https://github.com/pulp/pulpcore/discussions). Feel free to ask and answer questions, give feedback about your experience working with any aspect of Pulp. Introduce yourself, your project, what you're hoping to achieve with Pulp. We would be delighted to hear from you and understand how Pulp is helping.
+
```

Comment:
One random thought here. Are we able to moderate these discussions? Should we say, that we will always take the liberty to remove inappropriate posts?
I know, that we have chat logs of IRC anywhere, but I feel that this Discussions thing is by far more persistent.

Reply:
We have a code of conduct posted on the website?

Reply:
https://pulpproject.org/conduct/

Reply:
@mdellweg this actually appears as you write a discussion topic https://github.com/pulp/pulpcore/blob/e2ad45731e42d07a09a41aed254565baa153e535/CODE_OF_CONDUCT.md along the side of the new discussion topic!